src/base/bittorrent/torrentimpl.h

```diff
@@ -279,7 +279,9 @@ namespace BitTorrent
 
         void setAutoManaged(bool enable);
 
+        Path wantedActualPath(int index, const Path &path) const;
```

Comment:
I wonder if we should start making use of `[[nodiscard]]` here and there?
Any thoughts?